website/src/css/custom.css

```diff
@@ -40,14 +55,6 @@
   font-style: normal;
 }
```

Comment:
@yangshun should i remove fonts from **website/static/css/custom.css** too?

Reply:
Yes please